core/baseline-core-v1.0-psd01.md

```diff
@@ -566,195 +622,877 @@ Below we list and define the components of each layer. The detailed requirements
+
+In the context of this document, virtualized APIs are defined as a production sandbox for continuos integration testing and continuous deployment of APIs.
 
-This section describes mechanisms to ensure counterparties confidentiality and shielded private transactions.
+**[D12]**	A BPI Abstraction Layer SHOULD support a content delivery network (not applicable for a CCSM Abstraction Layer).
 
-## 5.1 Privacy
-Describes mechanisms ensuring that only information relevant to the transaction and pre-agreed by verified parties is used to its purpose.
+In the context of a BPI a content delivery network is a geographically distributed proxy server network providing high availability and delivery performance of content such as large data files or video streams.
 
-| Requirement ID|Requirement  |
-| :--- | :--- |
-| PRICON1 | something MUST something  |
-| PRICON2 | something SHOULD something |
+**[CR13]>[D12]** A CDN utilized in a BPI Abstraction Layer and operated by a 3rd party MUST support BPI user specific and time based content access control. 
 
+**[CR14]>[D12]** A CDN utilized in a BPI Abstraction Layer and operated by a 3rd party MUST support time based, automated content removal.
 
-## 5.2 Confidentiality
+**[R61]**	BPI Abstraction Layers MUST support integration with internal, as defined in [Section 5](#5-Middleware-Communication-and-Interoperability), and/or external BPI User identity access management (IAM) or identity provider (IdP) systems. 
 
-Describes the mechanisms ensuring that other parties (i.e parties outside of transaction) are prevented from accessing data that they are not authorized to access.
+See Figure 4 as to the meaning of an IdP in a BPI context, and Figure 5 in this document as to the meaning of external IAM and its interplay with BPI IAM discussed in Section 5(##5-Middleware-Communication-and-Interoperability).
+
+**[R62]**	BPI Abstraction Layers MUST support API delivery utilizing the service orchestration capabilities of the BPI Middleware Layer defined in [Section 5](#5-Middleware-Communication-and-Interoperability)
+
+**[R63]**	BPI Abstraction Layers MUST support facilitating the discovery and negotiation of capabilities and subsequent integration between a BPI and Legacy Systems/other BPIs as defined in [Section 5](#5-Middleware-Communication-and-Interoperability) (for BPI Abstraction Layer only).
+
+**[R64]**	BPI Abstraction Layers MUST support integration of CCSM specific transaction interfaces, transaction crafting, and CCSM specific smart contract management. 
+
+Smart Contract management comprises full lifecycle management from testing, initial deployments, update and deactivation (for CCSM Abstraction Layer only). 
+
+The Figure 6 below shows the reference architecture for a BPI or CCSM Abstraction Layer. 
+
+<figure>
+  <img
+  src="./images/Baseline-Fig2-BPI-Abstraction-Layer.png"
+  >
+  <figcaption>Figure 6: High-Level Reference Architecture of a BPI Layer.</figcaption>
```

Comment:
Please provide legends to explain colors

Reply:
Or all black and white if they are not grouped by color.

Reply:
b&w figure now